tools/run_tests/performance/templates/loadtest_template_prebuilt_all_languages.yaml

```diff
@@ -27,6 +27,19 @@ spec:
       - bash
       image: ${prebuilt_image_prefix}/csharp:${prebuilt_image_tag}
       name: main
+  - language: dotnet
+    pool: ${client_pool}
+    run:
+    - args:
+      - -c
+      # TODO(jtattermusch): why is the extra "--" in cmdline needed?
```

Comment:
@JamesNK  the need for `--` in the cmdline got me by surprised (since the QpsWorker binary is created by "dotnet publish").